yt_dlp/extractor/lsm.py

```diff
@@ -0,0 +1,282 @@
+            'title': 'D23-6000-105_cetstud',
+            'thumbnail': 'https://store.cloudycdn.services/tmsp00060/assets/media/660858/placeholder1700589200.jpg',
+        }
+    }, {
+        'url': 'https://ltv.lsm.lv/embed?enablesdkjs=1&c=eyJpdiI6IncwVzZmUFk2MU12enVWK1I3SUcwQ1E9PSIsInZhbHVlIjoid3FhV29vamc3T2sxL1RaRmJ5Rm1GTXozU0o2dVczdUtLK0cwZEZJMDQ2a3ZIRG5DK2pneGlnbktBQy9uazVleHN6VXhxdWIweWNvcHRDSnlISlNYOHlVZ1lpcTUrcWZSTUZPQW14TVdkMW9aOUtRWVNDcFF4eWpHNGcrT0VZbUNFQStKQk91cGpndW9FVjJIa0lpbkh3PT0iLCJtYWMiOiIyZGI1NDJlMWRlM2QyMGNhOGEwYTM2MmNlN2JlOGRhY2QyYjdkMmEzN2RlOTEzYTVkNzI1ODlhZDlhZjU4MjQ2IiwidGFnIjoiIn0=',
+        'md5': 'a1711e190fe680fdb68fd8413b378e87',
+        'info_dict': {
+            'id': 'wUnFArIPDSY',
+            'ext': 'mp4',
+            'uploader': 'LTV_16plus',
+            'release_date': '20220514',
+            'channel_url': 'https://www.youtube.com/channel/UCNMrnafwXD2XKeeQOyfkFCw',
+            'view_count': int,
+            'availability': 'public',
+            'thumbnail': 'https://i.ytimg.com/vi/wUnFArIPDSY/maxresdefault.jpg',
+            'release_timestamp': 1652544074,
+            'title': 'EIROVĪZIJA SALĀTOS',
+            'live_status': 'was_live',
+            'uploader_id': '@LTV16plus',
+            'comment_count': int,
+            'channel_id': 'UCNMrnafwXD2XKeeQOyfkFCw',
+            'channel_follower_count': int,
+            'categories': ['Entertainment'],
+            'duration': 5269,
+            'upload_date': '20220514',
+            'age_limit': 0,
+            'channel': 'LTV_16plus',
+            'playable_in_embed': True,
+            'tags': [],
+            'uploader_url': 'https://www.youtube.com/@LTV16plus',
+            'like_count': int,
+            'description': 'md5:7ff0c42ba971e3c13e4b8a2ff03b70b5',
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = urllib.parse.unquote(self._match_id(url))
+        webpage = self._download_webpage(url, video_id)
+        data = self._search_json(
+            r'window\.ltvEmbedPayload\s*=', webpage, 'embed json', video_id)
+        embed_type = traverse_obj(data, ('source', 'name', {str}))
+
+        if embed_type == 'telia':
+            ie_key = 'CloudyCDN'
+            embed_url = traverse_obj(data, ('source', 'embed_url', {url_or_none}))
+        elif embed_type == 'youtube':
+            ie_key = 'Youtube'
+            embed_url = traverse_obj(data, ('source', 'id', {str}))
+        else:
+            raise ExtractorError(f'Unsupported embed type {embed_type!r}')
+
+        return self.url_result(
+            embed_url, ie_key, video_id, **traverse_obj(data, {
+                'title': ('parentInfo', 'title'),
+                'duration': ('parentInfo', 'duration', {int_or_none}),
+                'thumbnail': ('source', 'poster', {url_or_none}),
+            }))
+
+
+class LSMReplayIE(InfoExtractor):
+    _VALID_URL = r'https?://replay\.lsm\.lv/.*/(?:ieraksts|statja)/[^/?#]+/(?P<id>\d+)'
```

Comment:
could we do this instead?
```suggestion
    _VALID_URL = r'https?://replay\.lsm\.lv/(?:[^/?#]+/)?(?:ieraksts|statja)/[^/?#]+/(?P<id>\d+)'
```